src/lib/import-export/import/handlers/backup-handler-zip.ts

```diff
@@ -1,41 +1,92 @@
 import { EventEmitter } from 'events';
-import AdmZip from 'adm-zip';
+import * as fs from 'fs';
+import * as path from 'path';
+import { promisify } from 'util';
+import * as fse from 'fs-extra';
+import * as yauzl from 'yauzl';
```

Comment:
```suggestion
import fs from 'fs';
import path from 'path';
import { promisify } from 'util';
import fse from 'fs-extra';
import yauzl from 'yauzl';
```
It's my fault that ESLint's `import/no-named-as-default-member` rule is enabled (happened in https://github.com/Automattic/studio/pull/825). I would advocate disabling that rule in `.eslintrc.json`. This import syntax looks foreign, and I struggle to see how it improves things for us in practice.